README.md

```diff
@@ -19,12 +19,46 @@ In order to create a trigger that reacts when messages are posted to a Message H
 
 While this list of parameters may seem daunting, they can be automatically set for you by using the package refresh CLI command:
 
-```wsk package refresh```
+1. Create an instance of Message Hub service under your current organization and space that you are using for OpenWhisk.
+
+2. Verify that the the topic you want to listen it already exists in Message Hub or create a new topic to listen for messages, like `mytopic`.
```

Comment:
- "to listen ~it~ **to**"
- "or create a new topic, ~to listen for messages, like~ for example `mytopic`